gradle.properties

```
@@ -1,2 +1,2 @@
 org.gradle.caching=true
-version=0.9.23
+version=0.9.24
```

```suggestion
version=0.9.25
```

This version is already being used